next.config.js

```diff
@@ -3,7 +3,20 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
 });
 
 /** @type {import("next/dist/lib/config-shared").Header['headers']} */
+
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-inline' *.googletagmanager.com 'unsafe-eval';
+  style-src 'self' 'unsafe-inline';
+  img-src 'self' blob: data: https:;
+  frame-ancestors 'none';
+`;
+
```

Comment:
```suggestion
const ContentSecurityPolicy = `
  default-src 'self';
  script-src 'self' 'unsafe-inline' *.googletagmanager.com 'unsafe-eval';
  style-src 'self' 'unsafe-inline';
  img-src 'self' blob: data: https:;
  frame-ancestors 'none';
`;

/** @type {import("next/dist/lib/config-shared").Header['headers']} */
```

Don't split the JSDoc block from its variable. It is there for a reason.

Reply:
sorry, and already fixed